lib/lunchbot/lunchbot_data/order.ex

```diff
@@ -22,7 +22,13 @@ defmodule Lunchbot.LunchbotData.Order do
 
   def get_total(%Lunchbot.LunchbotData.Order{} = order) do
     # total price of an Order the the price of all the items in the order.
-    Enum.map(order.order_items, &Lunchbot.LunchbotData.OrderItem.get_total/1)
-    |> Enum.sum()
```

Comment:
I kind of expected a simpler calculation here
```
|> Filter.nils
|> Enum.sum()
```

Reply:
Discussed in standup; not looking to soley filter out nils, looking to return nil overall.